src/js/view/controls/templates/rightclick.js

```diff
@@ -3,7 +3,7 @@ export default (menu, localization) => {
     const menuItems = items.map(item => rightClickItem(item, localization));
 
     return (
-        `<div class="jw-rightclick jw-modal jw-reset">` +
+        `<div class="jw-rightclick jw-reset">` +
```

Comment:
are we sharing any styles with modal now? i know info-modal was the basis for implementation, just curious if this fix is causing repetition

Reply:
by sharing i mean are there any styles that could be shared that no longer are because of the nature of this change

Reply:
there aren't any. Originally it was my understanding that the rightclick menu was meant to be moved into modal styling, so I was using that class to share the common styles between various modal types.

Reply:
oh nice!